src/2025h1/const-trait.md

```diff
@@ -0,0 +1,101 @@
+# Prepare const traits for stabilization
+
+| Metadata         |                          |
+|------------------|--------------------------|
+| Point of contact | @oli-obk                 |
+| Teams            | <!-- TEAMS WITH ASKS --> |
+| Task owners      | <!-- TASK OWNERS -->     |
+| Status           | Proposed                 |
+
+## Summary
+
+Prepare `const Trait` bounds for stabilization.
+
+## Motivation
+
+`const fn` on stable are unable to invoke trait methods, limiting their usefulness. After years of experimentation, the compiler now has a promising implentation of `const traits` and significant parts of the stdlib have been updated to use it. However, the feature is still firmly in experimental territory: there has never been an RFC describing its syntax.
```

Comment:
We haven't done the sweeping libstd impl yet, as the churn on any lang changes would be too large. But key parts have been done and lots of no_core tests in the test suite

Reply:
also I couldn't help but notice `implentation`

```suggestion
`const fn` on stable are unable to invoke trait methods, limiting their usefulness. After years of experimentation, the compiler now has a promising implementation of `const traits` and significant parts of the stdlib have been updated to use it. However, the feature is still firmly in experimental territory: there has never been an RFC describing its syntax.
```

but maybe that should say "key parts of the stdlib" ig